src/xdocs/config_imports.xml

```diff
@@ -49,9 +56,7 @@
           <tr>
             <td>excludes</td>
             <td>
-              packages where star imports are allowed. Note that this property
-              is not recursive, subpackages of excluded packages are not
-              automatically excluded.
+              Specify packages where star imports are allowed.
```

Comment:
I am not sure that moving details of property to Notes is good idea. User will read Notes only when he is in trouble to configure, and force himself to read whole doc.

@rnveach , please pay attention, if you agree, let's request a change.

Reply:
We already did this in 1 or more previous checks. The problem with keeping it in the property is it gets populated to the class javadoc, the field javadoc and the setter javadoc.
I thought we agreed that property description should be short and any specific details should be in notes or some other description.

Reply:
Lets be honest, almost nobody read our javadoc, most our readers are html documentation readers.
Nobody will suffer with extra lines in javadoc, but xdoc/html will benefit a lot and users will do less mistakes as they will see all nuances in one place in description of property, rather than later on after wasting a lot of time and forced to read whole doc.

Reply:
try to make experiment on your colleges, to get to know how much time it would resolve nuance of unexpected behavior take and eventually find NOTE section. Ideally configuration should be properly done right after reading of property description.

BTW, ... we can make `see Notes for more details on behavior` sentence to keep description short, if it is still a concern. 

I am merging this PR, as any enhancements can be done other PRs.